app/main.py

```diff
@@ -1,2 +1,85 @@
+from typing import Any, Hashable
+
+
 class Dictionary:
-    pass
+    load_factor = 2 / 3
+
+    def __init__(self) -> None:
+        self.length = 0
+        self.capacity = 8
+        self.hash_table = [None] * self.capacity
+
+    def __setitem__(
+            self,
+            key: Hashable,
+            value: Any
+    ) -> None:
+        key_hash = hash(key)
+        items = [key, key_hash, value]
```

Comment:
It would make sense to implement a separate Node class as per task requirements. It will help to better structure your implementation. 

> Also, don't forget to store (key, hash, value) as a node in the hash table. A hash table may be stored as a simple list of nodes.